create_object.yml

```diff
@@ -7,7 +7,8 @@
 # Usage:
 #
 #   $ bin/ansible-playbook create_object.yml \
-#       -e "object_template=templates/openshift/edxapp/job/import_demo_course.yml.j2"
+#       -e "object_template=templates/openshift/edxapp/job/import_demo_course.yml.j2" \
+#       -e "deployment_stamp=2018-06-07-15-41-31-838798865"
```

Comment:
don't forget to change the stamp format in this docstring

Reply:
Done